### ISSUE TITLE
Should AJD template be updated to match BEM standards?

### ISSUE THREAD

**Issue by Brockenstein** [radancyco/tb-ajd-template#9]
@jkmahoney once BEM standards are set we should revisit the AJD template and determine if the AJD needs updating to match BEM standards and the implications of changing the classes, such as JS changes, how it would affect live sites and so forth.

**Comment by jkmahoney** [comment]
@Brockenstein, I agree. Once we finalize BEM standards the AJD template should be updated to reflect those changes. Any concerns with changes to the AJD template should be brought up when we discuss our BEM standards to see if we need to address those issues.
